pkgs/top-level/python-packages.nix

```
@@ -1694,6 +1694,8 @@ in {
 
   bumps = callPackage ../development/python-modules/bumps {};
 
+  bx-python = disabledIf (!isPy3k) (callPackage ../development/python-modules/bx-python { });
```

move into expression, see other comment. Also explicitly pass top-level packages to avoid python-package additions shadowing them.
```suggestion
  bx-python = callPackage ../development/python-modules/bx-python {
    inherit (pkgs) zlib;
  };
```